README.md

```diff
@@ -16,56 +48,4 @@ You can use a tool like [nvm](https://github.com/nvm-sh/nvm) to get the version
 
 1. Clone the project with `git clone git@github.com:Code-For-Chicago/Voma-frontend.git`
 2. Install dependencies with `npm i` (see [npm](https://www.npmjs.com/))
-3. Start development server with `npm start`. App will be served at [http://localhost:3000/](http://localhost:3000/)
-
-## NPM Scripts
-
-In the project directory, you can run the following scripts:
-
-### `npm start`
-
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm build`
-
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-### `npm lint`
-
-Run [ESLint](https://eslint.org/) to check for any problems in your JavaScript code.
-
-### `npm lint:css`
-
-Run [stylelint](https://stylelint.io/) to check for any problems in your CSS code.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
```

Comment:
I believe we should leave this in. It a nice bit of information for a quick start on contributing.

Reply:
All we need at the moment is 'npm start' which is mentioned earlier. 

We can adjust the docs as we add more scripts.

Reply:
I could go either way - Juliette suggested removing it in Slack.

Might be a bit much for the README - but we could have a second file for setup instructions?  I know that's on our list of things to do in the short term.  Just an idea.